ssl/ssl_rsa.c

```diff
@@ -424,6 +424,9 @@ static int use_certificate_chain_file(SSL_CTX *ctx, SSL *ssl, const char *file)
     void *passwd_callback_userdata;
     SSL_CTX *real_ctx = (ssl == NULL) ? ctx : ssl->ctx;
 
+    if (ctx == NULL && ssl == NULL)
```

Comment:
I assume this is a theoretical case as opposed to something that actually happens.

Reply:
It shouldn't happen.  It's a very cheap (if not free) check.